domain/schema/model/sql/0033-operation.sql

```diff
@@ -127,3 +127,14 @@ CREATE TABLE operation_task_log (
     FOREIGN KEY (task_uuid)
     REFERENCES operation_task (uuid)
 );
+
+-- operation_parameter holds the parameters passed to an operation.
+CREATE TABLE operation_parameter (
+    operation_uuid TEXT NOT NULL,
+    "key" TEXT NOT NULL,
+    value TEXT NOT NULL,
+    PRIMARY KEY (operation_uuid, "key"),
+    CONSTRAINT fk_operation_uuid
+    FOREIGN KEY (operation_uuid)
+    REFERENCES operation (uuid)
+);
```

Comment:
Looking at 3.6, these look immutable, you could if you so desired add a trigger for that and  the operations, but it might be overkill.

Reply:
Yeah I saw, I was thinking to enforce that as business logic (code) instead of DDL. What do you think?